### PR TITLE
Oracle ODBC DSN configuration from external file

### PR DESCRIPTION
**Pull request by 5380harshjain**
@robe070 
Here I've made the changes as per your feedback
1.) I'm assuming that the initial Host value present in tnsnames.ora will not be static ("ora150050.cnyed5gpqwey.us-east-1.rds.amazonaws.com")
2.) Assuming the above, I've replaced the complete line with the required line containing the lansa version.

Please review it, Rob.

### CODE REVIEW

File: Scripts/configure-ODBC-DSN.ps1 [hunk 1]
```diff
@@ -5,31 +5,52 @@
...
 $DSNNames = (Get-OdbcDsn).Name
 
-# AZURESQL
 
-Write-Host("Updating Server name and database name for AZURESQL")
+# AZURESQL
 
 if($DSNNames -contains "AZURESQL" -and (Get-OdbcDsn -Name "AZURESQL").Platform -eq "32-bit"){
+
+    Write-Host("Updating Server name and database name for AZURESQL")
```
Comment (robe070) [inline Scripts/configure-ODBC-DSN.ps1]: You MUST output the values you are updating the ODBC DSN with.

File: Scripts/configure-ODBC-DSN.ps1 [hunk 2]
```diff
@@ -5,31 +5,52 @@
...
 }
 
-# MYSQL
 
-Write-Host("Updating Server name and database name for MYSQL")
+# MYSQL
 
 if($DSNNames -contains "MYSQL" -and (Get-OdbcDsn -Name "MYSQL").Platform -eq "32-bit"){
+
+    Write-Host("Updating Server name and database name for MYSQL")
```
Comment (robe070) [inline Scripts/configure-ODBC-DSN.ps1]: You MUST output the values you are updating the ODBC DSN with.

File: Scripts/configure-ODBC-DSN.ps1 [hunk 3]
```diff
@@ -5,31 +5,52 @@
...
 
+
 # Oracle
 
-Write-Host("Updating Server name for Oracle")
+if($DSNNames -contains "ora19cdb" -and (Get-OdbcDsn -Name "ora19cdb").Platform -eq "32-bit"){
+
+    $TNSNamesPath = "C:\oracle\instantclient_21_6\network\admin\tnsnames.ora"
+    $Content = Get-Content $TNSNamesPath
+    $oraServer = "ora" + $lansaVersion + ".cnyed5gpqwey.us-east-1.rds.amazonaws.com"
```
Comment (robe070) [inline Scripts/configure-ODBC-DSN.ps1:28]: You need to derive the endpoint from AWS.

**Review by robe070 — CHANGES_REQUESTED** [review]

File: Scripts/configure-ODBC-DSN.ps1 (hunk 1 above)
Comment (5380harshjain) [reply Scripts/configure-ODBC-DSN.ps1]: Done.

File: Scripts/configure-ODBC-DSN.ps1 (hunk 2 above)
Comment (5380harshjain) [reply Scripts/configure-ODBC-DSN.ps1]: Done.

File: Scripts/configure-ODBC-DSN.ps1 (hunk 3 above)
Comment (5380harshjain) [reply Scripts/configure-ODBC-DSN.ps1:28]: Not required anymore.

**Review by robe070 — APPROVED** [review]

**Review by robe070 — CHANGES_REQUESTED** [review]
Please resolve the conflicts

**Comment by 5380harshjain** [comment]
> Please resolve the conflicts

Done.

**Pull request merged**